pkg/sloop/storemanager/stats_test.go

```diff
@@ -0,0 +1,31 @@
+package storemanager
+
+import (
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+	"path"
+	"testing"
+)
+
+func Test_GetDirSizeRecursive(t *testing.T) {
+	fs := afero.Afero{Fs: afero.NewMemMapFs()}
+	fs.MkdirAll(someDir, 0700)
+	// 3 vlog files
+	fs.WriteFile(path.Join(someDir, "000010.vlog"), []byte("a"), 0700)
+	fs.WriteFile(path.Join(someDir, "000011.vlog"), []byte("aa"), 0700)
+	fs.WriteFile(path.Join(someDir, "000012.vlog"), []byte("aaaaa"), 0700)
+	// 4 sst files
+	fs.WriteFile(path.Join(someDir, "000070.sst"), []byte("zzzzzz"), 0700)
+	fs.WriteFile(path.Join(someDir, "000071.sst"), []byte("zzzzzzz"), 0700)
+	fs.WriteFile(path.Join(someDir, "000072.sst"), []byte("zzzzzzzz"), 0700)
+	fs.WriteFile(path.Join(someDir, "000073.sst"), []byte("zzzzzzzzz"), 0700)
+	// Other
+	fs.WriteFile(path.Join(someDir, "KEYREGISTRY"), []byte("u"), 0700)
+	fs.WriteFile(path.Join(someDir, "MANIFEST"), []byte("u"), 0700)
```

Comment:
this test should probably also include some unexpected files and perhaps some depth to the folder structure

Reply:
added